package.json

```diff
@@ -94,6 +99,10 @@
     "orbit-db": "git://github.com/orbitdb/orbit-db.git#main",
     "orbit-db-store": "^4.1.0",
     "peer-id": "^0.14.0",
+    "prettier-config-standard": "^1.0.1",
+    "protons": "^2.0.0",
+    "random-timestamps": "^1.0.1",
+    "simple-git": "^2.24.0",
```

Comment:
I am pretty sure that we don't need this. I removed this and few other packages. It's possible that you left it by mistake while resolving git conflicts.